### ISSUE TITLE
Removed files in wheel

### ISSUE THREAD

**Issue by lambdalisue** [ronf/asyncssh#247]
I noticed that the wheel file of asyncssh 2.1.0 on [pypi](https://pypi.org/project/asyncssh/#files) contains the following removed directories/files.

<img width="426" alt="asyncssh-2 1 0-py3-none-any whl 2020-01-15 15-26-29" src="https://user-images.githubusercontent.com/546312/72410298-729c4280-37ab-11ea-9d01-a4b20d670763.png">

So that users who installed asyncssh via wheel see quite confusing message when they accidentally tried to import `asyncssh.crypto.pyca` or whatever.

```
Python 3.7.6 (default, Dec 25 2019, 14:48:36)
[Clang 11.0.0 (clang-1100.0.33.8)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import asyncssh.crypto.pyca
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/alisue/.anyenv/envs/pyenv/versions/3.7.6/lib/python3.7/site-packages/asyncssh/crypto/pyca/__init__.py", line 15, in <module>
    from . import cipher
  File "/Users/alisue/.anyenv/envs/pyenv/versions/3.7.6/lib/python3.7/site-packages/asyncssh/crypto/pyca/cipher.py", line 144, in <module>
    CipherFactory(_cipher, _mode))
TypeError: register_cipher() missing 1 required positional argument: 'block_size'
>>>
```

**Comment by lambdalisue** [comment]
Additionally, this cause issue for using [`pkgutil.walk_packages`](https://docs.python.org/3/library/pkgutil.html#pkgutil.walk_packages) while the error is not `ImportError` so that the function propagate that error.

**Comment by ronf** [comment]
Huh! The files in those directories are incredibly outdated (and in fact were removed from the source tree years ago), and the associated packages are NOT listed in setup.py, so I'm surprised the wheel-generation is still picking them up.

It looks to me like the "build" directory (more specifically build/lib) is not being cleared out when a build is done, and these files are being picked up from there. I'll make sure to clear this out the next time I produce a wheel. Thanks for the heads-up!

**Comment by ronf** [comment]
Looking more closely, I see this also applies to other files in the asyncssh and crypto directories which were later renamed or removed. I'll make sure those are cleaned out as well in the next wheel.

**Comment by ronf** [comment]
This problem should now be corrected with the newly released AsyncSSH 2.2.0 wheel. Thanks for the report!
